src/evm_context.rs

```diff
@@ -109,7 +109,7 @@ context_variables! {
     number: Type::Default; None,
     origin: Type::Default; None,
     timestamp: Type::Default; None,
-    memory: Type::Constant("(Array (_ BitVec 256) (_ BitVec 8))"); None,
+    memory: Type::Constant("(Array (_ BitVec 256) (_ BitVec 8))"); Some("((as const (Array (_ BitVec 256) (_ BitVec 8))) #x00)".to_string()),
```

Comment:
OMG how to do this in the new framework?
Do we need a new smt function `as_const`?